### ISSUE TITLE
General Feedback - Content harmonization for Assay and Instrument 

### ISSUE THREAD

**Issue by teslajoy** [NIH-NCPI/ncpi-fhir-ig-2#80]
### What were you reviewing?
https://nih-ncpi.github.io/ncpi-fhir-ig-2/CodeSystem-assay-strategy-cs.html
https://nih-ncpi.github.io/ncpi-fhir-ig-2/CodeSystem-platform-instrument-cs.html

Content harmonization: 
Assay types and instruments may be further extended for standard types. 
https://nih-ncpi.github.io/ncpi-fhir-ig-2/CodeSystem-assay-strategy-cs.html
https://nih-ncpi.github.io/ncpi-fhir-ig-2/CodeSystem-platform-instrument-cs.html

### Suggested Improvements:
Example:  
RNA-Seq
scRNA-Seq
scATAC-Seq
WXS
RPPA
Cycif 
IMC
CODEX
MIBI
MIHC
H&E
…

**Comment by RobertJCarroll** [comment]
For this, we'd like to identify a suitable vocabulary or ontology that incorporates structured terms.

**Comment by calkinsh** [comment]
@RobertJCarroll I agree with that! I think when we reviewed this all last summer, EDAM was the best option we found.  I did a quick search again and i dont see anything new or improved in the intervening time, so I think that EDAM subset is still the best bet.

**Comment by RobertJCarroll** [comment]
I just stumbled across this list from dbGaP: https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/GetMolecularDataTypes.cgi Might be good to consider if we have coverage for these known concepts.

**Comment by RobertJCarroll** [comment]
We had discussed [OBI assays](https://www.ebi.ac.uk/ols4/ontologies/obi/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FOBI_0000070?lang=en) instead, how do we feel about that versus EDAM?

**Comment by RobertJCarroll** [comment]
A few examples:
whole genome sequencing assay OBI:0002117
exome sequencing assay OBI:0002118
genotyping by array assay OBI:0001274
bulk RNA-Seq assay OBI:0003090
polyA-selected RNA sequencing assay OBI:0002571
single-cell RNA sequencing assay OBI:0002631
single-cell ATAC-seq OBI:0002764

**Comment by RobertJCarroll** [comment]
Closed per #152
